.github/workflows/build.yml

```diff
@@ -10,15 +10,24 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        emacs-version:
+          - 26.3
+          - 27.2
+          - 28.2
+          - snapshot
 
     steps:
       - uses: actions/checkout@v3
 
       - name: Setup Emacs
-        uses: purcell/setup-emacs@master
+        uses: jcs090218/setup-emacs@master
```

Comment:
Is there a reason you changed the setup action. I appreciate the fact you're building your own custom action, but purcell/setup-emacs seems in wider use ( currently ). Happy to change if it there's a good reason.

Reply:
purcell's action don't support windows, and I don't think it will ever going to support it.